app/controllers/invite_requests_controller.rb

```diff
@@ -49,7 +50,7 @@ def manage
     end
   end
 
-  def reorder
+  def reorder_list
```

Comment:
Why was this marked as resolved?

Reply:
( As when I changed it all I ended up going though a loop of getting the brakeman errors back ) . I will do the change as an isolated commit and see if that is still the case.

Reply:
Done